source/Octopus.Client/Exceptions/InvalidSpacesLimitationParametersException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Octopus.Client.Exceptions
+{
+    public class InvalidSpacesLimitationParametersException: Exception
+    {
+        public InvalidSpacesLimitationParametersException(): base("The new spaceIds can only be a subset of the previously defined spaceIds")
```

Comment:
I think the word `Parameters` here is redundant. Let's just make it `InvalidSpacesLimitationException`